app/screens/interactive_dialog/interactive_dialog.js

```diff
@@ -190,6 +213,12 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
         container: {
             backgroundColor: changeOpacity(theme.centerChannelColor, 0.03),
         },
+        errorContainer: {
+            marginTop: 15,
+            marginLeft: 15,
```

Comment:
These numbers were chosen to match the alignment of the field labels.  I wonder if there is a more general way to make the labels align.

Reply:
There is but apparently not applicable in this case?see [react native documentation](http://facebook.github.io/react-native/docs/flexbox) to understand more about the layout system.

Reply:
Thanks for the link, I will look more into it.